website/docs/r/terraform_version.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_terraform_version"
+sidebar_current: "docs-resource-tfe-terraform-version-x"
+description: |-
+  Manages Terraform versions
+---
+
+# tfe_terraform_version"
+
+Manages Terraform versions.
+
+## Example Usage
+
+Basic Usage:
+
+```hcl
+resource "tfe_terraform_version" "test" {
+  version = "99.99.99"
```

Comment:
Maybe we shouldn't encourage this type of nonsemantic versioning. How about

`version = "1.1.2-custom"`